packages/@react-spectrum/s2/intl/fr-FR.json

```diff
@@ -22,6 +23,6 @@
   "table.sortDescending": "Trier par ordre décroissant",
   "tag.actions": "Actions",
   "tag.hideButtonLabel": "Afficher moins",
-  "tag.noTags": "Aucun",
+  "tag.noTags": "Aucun(e)",
```

Comment:
?

Reply:
I'm reverting this, change leveraged from another app, but no linguistic reason for change in React-Spectrum